### ISSUE TITLE
Problem with color of info icon

### ISSUE THREAD

**Issue by Szymon20000** [commons-app/apps-android-commons#2940]
![image](https://user-images.githubusercontent.com/12784455/57249260-c2cfdc00-7044-11e9-9514-0ed39dfefabe.png)

Info icon is not visible because it's always white(skip_this_image_button).  
Android 6.0.1
Samsung note 4.

**Comment by neslihanturan** [comment]
This issue is beginner friendly

**Comment by madhurijain97** [comment]
![sample](https://user-images.githubusercontent.com/16744716/57332077-ec354a00-70ce-11e9-8502-ef9f46e16be3.jpg)

I am able to see the blue info icon on Android 8.
Is this issue specific to Android 6?

The drawable_tint_color is set to blue in the activity layout, so it should ideally be showing in blue color. However, if the issue still persists, we can change the color of the drawable icon itself?
I am a beginner. Please correct me if I am missing out something.

**Comment by neslihanturan** [comment]
Oh, thanks for reporting this @madhurijain97 . Can you please make sure tried with both dark and light theme (from settings).

**Comment by madhurijain97** [comment]
![sample_dark](https://user-images.githubusercontent.com/16744716/57334552-ee020c00-70d4-11e9-9d27-901d01b545d0.jpg)

Yes, I tried with both.
The blue info icon shows up in the dark theme too.

**Comment by neslihanturan** [comment]
I can't reproduce the issue either,
![image](https://user-images.githubusercontent.com/3127881/57357442-a4203100-717b-11e9-88f4-52179660c68a.png)

@Szymon20000 can you share the about page screenshot of the device you reproduce this issue?

**Comment by Vicropht** [comment]
This seems to be an android 6 specific issue, everything fine here (Huawei WAS-LX1A Android 8 used)

**Comment by thetimeheist76** [comment]
I am a beginner and if this issue is still open, I would like to work on it.

**Comment by thetimeheist76** [comment]
If this issue is still open I would like to work on it.

**Comment by neslihanturan** [comment]
This is open but someone else already works on this. So please check for a new issue. Looking for beginner friendly label and asking if anyone else working on it is a good practice for newcomers :) Additionally you can check out our documentation before starting.

**Comment by nicolas-raoul** [comment]
@thetimeheist76 Did you manage to reproduce the issue? If yes feel free to start working on it, thanks a lot :-)

**Comment by nicolas-raoul** [comment]
@thetimeheist76 Sorry I had not seen Neslihan's comment! And the issue should have been tagged as "assigned".

**Comment by thetimeheist76** [comment]
@nicolas-raoul  yes I was able to reproduce the issue. I first tried with Android 7 and it did not raise the issue. Later I tried with Android 5 and was able to reproduce. I'll now start working on it. Thanks a lot.

**Comment by thetimeheist76** [comment]
Was able to resolve the issue. This is how it now looks
![WhatsApp Image 2019-07-26 at 10 50 46 PM](https://user-images.githubusercontent.com/53221361/61970013-7ff52300-aff9-11e9-9e13-c9a8156a73c8.jpeg)
![WhatsApp Image 2019-07-24 at 11 50 00 PM](https://user-images.githubusercontent.com/53221361/61970014-7ff52300-aff9-11e9-801e-b541f170108f.jpeg)

**Comment by thetimeheist76** [comment]
Was able to resolve the issue. This is how it looks now. Wasn't able to get rid off 
![light theme](https://user-images.githubusercontent.com/53221361/61970358-407b0680-affa-11e9-9fa4-b2ad8de8bb02.jpeg)
![WhatsApp Image 2019-07-26 at 10 50 46 PM](https://user-images.githubusercontent.com/53221361/61970367-44a72400-affa-11e9-9824-76b484305be9.jpeg)

the blue background around the icon though.

**Comment by thetimeheist76** [comment]
Was able to fix the issue. This is how it looks now.
![img2](https://user-images.githubusercontent.com/53221361/61972321-f0527300-affe-11e9-81a5-0dae8d5fcf59.jpeg)
![img3](https://user-images.githubusercontent.com/53221361/61972324-f0eb0980-affe-11e9-9dc4-f9c1b33b438a.jpeg)

**Comment by neslihanturan** [comment]
Oh okay, seems like a second PR came about this issue. @PavelAplevich do you still working on this issue?

**Comment by Kushagra2000** [comment]
Is this issue still open?

**Comment by nicolas-raoul** [comment]
The issue has been fixed. Thanks all! :-)
